File: Lab_One/Rectangle.cpp
#pragma once
#include <iostream>

double rectangleSquare(double length, double width) {
	return length * width;
}

double rectanglePerimeter(double length, double width) {
	return 2 * length + 2 * width;
}

double rectangleDiagonal(double length, double width) {
	return sqrt(length*length + width*width);
}

File: kotlin/src/main/kotlin/Rectangle.kt
package org.example

import kotlin.math.sqrt

fun rectangleSquare(width : Double , length : Double)  = width*length
fun rectanglePerimeter(width : Double , length : Double) = (width+length)*2
fun rectangleDiagonal(width : Double , length : Double) = sqrt(length*length+width*width)